### ISSUE TITLE
Versioning Information

### ISSUE THREAD

**Issue by rkhamis** [threefoldtech/tfchain_explorer#106]
show version information on the explorer

should show:
  - version of explorer
  - TFChain version
  - grid proxy version
  - in node view, node says version of zos running

**Comment by RafyAmgadBenjamin** [comment]
grid proxy version depends on:
https://github.com/threefoldtech/tfgridclient_proxy/issues/55

**Comment by RafyAmgadBenjamin** [comment]
i am able to get now `grid proxy`, `explorer version` and `zos version `
still blocked on this
https://github.com/threefoldtech/tfchain/issues/155

**Comment by xmonader** [comment]
Let's go without tfchain version

**Comment by RafyAmgadBenjamin** [comment]
separated tfchain version feature in a separate issue
https://github.com/threefoldtech/tfchain_explorer/issues/113

**Comment by RafyAmgadBenjamin** [comment]
### Verified 
### Branch
`master`
### Tag
`v3.0.0-rc14`

- Explorer vesion and Grid proxy version
![image](https://user-images.githubusercontent.com/46320202/147669967-1f2c2c94-6194-4597-96e2-42087cf646ab.png)
- ZOS version
![image](https://user-images.githubusercontent.com/46320202/147670004-afcc2380-19ce-4146-8825-9cb2144d0b0a.png)
